cvat/apps/dashboard/static/dashboard/js/dashboard.js

```diff
@@ -207,20 +250,47 @@ class TaskView {
 
 
 class DashboardView {
-    constructor(metaData, taskData, annotationFormats) {
+    constructor(metaData, taskData, annotationFormats, exportFormats) {
         this._dashboardList = taskData.results;
         this._maxUploadSize = metaData.max_upload_size;
         this._maxUploadCount = metaData.max_upload_count;
         this._baseURL = metaData.base_url;
         this._sharePath = metaData.share_path;
         this._params = {};
         this._annotationFormats = annotationFormats;
+        this._exportFormats = [];
 
+        this._setupExportFormats(exportFormats);
         this._setupList();
         this._setupTaskSearch();
         this._setupCreateDialog();
     }
 
+    _setupExportFormats(availableFormats) {
```

Comment:
@zhiltsov-max ,

Do you think the server should incapculate the information below? Can we send a json from our server which already has the information?
```
{
    formats: [
        {
            tag: 'datumaro_project',
            name: "Dataset repository",
            is_public: false,
            is_default: true,
        },
        {
            tag: 'coco',
            name: "MS COCO",
            url: "",
            is_public: true,
            is_default: false,
        },
    ], 
}
```

Reply:
Technically, I like this idea, but this whole structure looks like client/view logic. What UI pattern do we use?